test/extensions/filters/network/http_connection_manager/config_test_base.h

```diff
@@ -0,0 +1,66 @@
+#include "envoy/config/core/v3/base.pb.h"
+#include "envoy/config/trace/v3/http_tracer.pb.h"
+#include "envoy/config/trace/v3/opencensus.pb.h"
+#include "envoy/config/trace/v3/zipkin.pb.h"
+#include "envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.pb.h"
+#include "envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.pb.validate.h"
+#include "envoy/server/request_id_extension_config.h"
+#include "envoy/type/v3/percent.pb.h"
+
+#include "common/buffer/buffer_impl.h"
+#include "common/filter/http/filter_config_discovery_impl.h"
+#include "common/http/date_provider_impl.h"
+#include "common/http/request_id_extension_uuid_impl.h"
+#include "common/network/address_impl.h"
+
+#include "extensions/filters/network/http_connection_manager/config.h"
+
+#include "test/extensions/filters/network/http_connection_manager/config.pb.h"
+#include "test/extensions/filters/network/http_connection_manager/config.pb.validate.h"
+#include "test/mocks/config/mocks.h"
+#include "test/mocks/http/mocks.h"
+#include "test/mocks/network/mocks.h"
+#include "test/mocks/server/factory_context.h"
+#include "test/test_common/printers.h"
+#include "test/test_common/registry.h"
+#include "test/test_common/test_runtime.h"
+#include "test/test_common/utility.h"
```

Comment:
There are a lot of headers here, might be nice to shrink them based on IWYU principles.

Reply:
Did my best, but it's pretty time-consuming and inaccurate doing this manually what with all the unseen nested dependencies.